generators/gae/index.js

```diff
@@ -66,12 +65,12 @@ module.exports = class extends BaseGenerator {
                         } else {
                             this.log(chalk.bold('\nInstalling App Engine Java SDK'));
                             this.log(`... Running: gcloud components install ${component} --quiet`);
-                            const child = exec(`gcloud components install ${component} --quiet`, {
-                                stdio: [process.stdin, process.stdout, process.stderr]
-                            });
-                            child.on('exit', code => {
-                                if (code !== 0) {
-                                    this.abort = true;
+                            exec(`gcloud components install ${component} --quiet`, err => {
+                                if (err) {
+                                    this.log.error(err);
+                                    done(
+                                        'App Engine Java SDK installation failed. \nPlease try to install manually; https://cloud.google.com/sdk/install'
```

Comment:
I think the error message here should prompt the user to run `gcloud components install` manually.

The `gcloud` sdk install was already checked previously.

Reply:
Thanks. I've changed it. Let me know if you need any further changes.